concrete/src/View/AbstractView.php

```diff
@@ -36,6 +39,12 @@ public static function getRequestInstance()
         return self::$requestInstance;
     }
 
+    /**
+     * @param mixed $mixed object to view
+     *
+     * @return void
```

Comment:
`@return void` is optional in general, and in constructors it should be omitted

Reply:
got it. good to know!